policy/modules/services/console_setup.if

```diff
@@ -0,0 +1,106 @@
+## <summary>console font and keymap setup program for debian</summary>
+
+########################################
+## <summary>
+##  Execute console-setup in the console_setup domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`console_setup_domtrans', `
```

Comment:
Please drop the `_` between "console" and "setup" here and in below interfaces, e.g. `consolesetup_domtrans()`.

Reply:
Should I rename the module too ?

Reply:
Yes.

Reply:
All other uses of `console_setup` too ? (`console_setup_t`, `console_setup_exec_t`, ...)